iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/model/AffMatchOrganization.java

```diff
@@ -78,6 +82,34 @@ public String getWebsiteUrl() {
         return websiteUrl;
     }
 
+    /**
+     * Alternative names of the organization, e.g. name
+     * in different languages than {@link #getName()}
+     */
+    public List<String> getAlternativeNames() {
+        return alternativeNames;
+    }
+
+
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Adds the name to the alternative names list
+     */
+    public void addAlternativeName(String alternativeName) {
+        this.alternativeNames.add(alternativeName);
+    }
+    
+    /**
+     * Returns the list with all organization names.
+     * That is main {@link #getName()} with all {@link #getAlternativeNames()} 
+     */
+    public List<String> getNames() {
+        List<String> names = Lists.newArrayList(name);
```

Comment:
I guess it will be better to allocate `ArrayList` with fixed capacity (`alternativeNames.size() +1`) and then add all the names. This will prevent from potential reallocation.